src/alfasim_sdk/_internal/constants.py

```diff
@@ -115,6 +115,12 @@ class EmulsionRelativeViscosityModelType(Enum):
     Mooney1951a = "mooney1951a"
     Mooney1951b = "mooney1951b"
     FromPlugin = "from_plugin"
```

Comment:
I prefer to put `FromPlugin` option as the last one. 
But I am not sure if it will break something with the `Enum`.

Reply:
I think this would require some treatment when loading old cases. I don't know if it is worthy to do such modification

Reply:
We save the "string" not some auto index or some thing like that.
That being said, adding new values (any where in the enum) will not affect loading old cases.
Changing values (the strings) or removing the entries will affect old cases using the old values.

Reply:
Nice! Thanks for the explanation @prusse-martin!

Reply:
But I also prefer `FromPlugin` kind placed apart from the others, like last or first.
Having it as first has the benefit of getting out of change sets, I will put as a possible problem with that that it will be the default value, but on case description no default is assumed for enums.
That will probably be an issue with alfasim, where every thing has defaults and possibly some enum use implicit defaults (the "first value") but that can be fixed without backward compatibility effects and easily detectable checking the status.

Reply:
You are right @prusse-martin. 

- [ ] Replace plugin entry